### ISSUE TITLE
Error in cwltoil 3.7.0

### ISSUE THREAD

**Issue by susannasiebert** [BD2KGenomics/toil#1666]
After fixing the `import os` error in `src/toil/batchSystems/lsf.py` (PR #1661) I get additional downstream errors:

```
  File "/usr/local/lib/python2.7/dist-packages/toil/batchSystems/lsf.py", line 220, in <genexpr>
    currentjobs = set(self.lsfJobIDs[x] for x in self.getIssuedBatchJobIDs())
KeyError: 0
```

**Comment by mr-c** [comment]
I also am experiencing this issue

**Comment by mr-c** [comment]
Seems to be caused by a failed job submission:

```
ipdb> ebi-cli-001.ebi.ac.uk 2017-05-17 02:47:31,790 Thread-201 DEBUG toil.batchSystems.lsf: BSUB: MEMLIMIT: Cannot exceed queue's hard limit(s). Job n.

Exception in thread Thread-201:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 811, in __bootstrap_inner
    self.run()
  File "/hps/nobackup/production/metagenomics/CWL/mcrusoe/toil-v3.8.0a/src/toil/batchSystems/lsf.py", line 117, in run
    lsfJobID = bsub(bsubline)
  File "/hps/nobackup/production/metagenomics/CWL/mcrusoe/toil-v3.8.0a/src/toil/batchSystems/lsf.py", line 51, in bsub
    result = int(liney.strip().split()[1].strip('<>'))
ValueError: invalid literal for int() with base 10: 'Cannot'
```

**Comment by mr-c** [comment]
There is a fix for this in #1573

**Comment by mr-c** [comment]
Fixed via #1573
